lld/ELF/Arch/PPC64.cpp

```diff
@@ -72,11 +72,7 @@ enum DFormOpcd {
   ADDI = 14
 };
 
-enum DSFormOpcd {
-  LD = 58,
-  LWA = 58,
-  STD = 62
-};
+enum DSFormOpcd { LD = 58, LWA = 58, STD = 62 };
```

Comment:
Could add a comma here for formatting purposes:
```suggestion
enum DSFormOpcd {
   LD = 58,
   LWA = 58,
   STD = 62,
 };
```